Add Spanish translation

MiguelProz opened the pull request:
He traducido todos los archivos JSON del directorio de localización al español de España:

Archivos traducidos:
[accounts.json] - Cuentas y gestión de usuarios
[ads.json] - Información sobre anuncios
[app.json] - Actualización de la aplicación
[auth.json] - Autenticación y login
[content.json] - Contenido de mods y addons
[enums.json] - Enumeraciones (versiones, cargadores)
[errors.json] - Mensajes de error
[general.json] - Términos generales
[instances.json] - Gestión de instancias
[java.json] - Configuración de Java
[library.json] - Biblioteca y colecciones
[logs.json] - Visualización de registros
[modals.json] - Ventanas modales y diálogos
[news.json] - Noticias y notas de cambios
[notifications.json] - Notificaciones del sistema
[onboarding.json] - Introducción para nuevos usuarios
[placeholders.json] - Textos de ejemplo
[search.json] - Búsqueda y filtros
[settings.json](vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) - Configuración general
[tasks.json](vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) - Tareas y operaciones
[tracking.json](vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) - Tracking y métricas
[ui.json](vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) - Elementos de interfaz
[window.json](vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) - Ventana y cierre de aplicación

Características de la traducción:
✅ Todas las traducciones están en español de España (Castilian Spanish)
✅ Se han respetado los espaciadores de plantilla ({{variable}})
✅ Se ha mantenido la estructura JSON intacta
✅ Se han adaptado los términos técnicos apropiadamente
✅ Se utilizó lenguaje formal y apropiado para una aplicación

MiguelProz committed:
Add Spanish localization for various components including instances, Java settings, library, logs, modals, news, notifications, onboarding, placeholders, search, settings, tasks, tracking, UI, and window messages.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/gorilla-devs/GDLauncher-Carbon?pullRequest=451) <br/>All committers have signed the CLA.

MiguelProz committed:
Añadir soporte para el idioma español en el archivo de localización

MiguelProz committed:
Merge branch 'develop' into develop

MiguelProz committed:
Merge branch 'develop' into develop

fulviodenza commented on packages/i18n/locale/spanish/settings.json:
```diff
@@ -0,0 +1,80 @@
+  "_trn_show_window_close_warning_title": "Mostrar advertencia de cierre de ventana al iniciar el juego",
+  "_trn_show_window_close_warning_text": "Muestra una advertencia al cerrar la ventana mientras se está ejecutando un juego.",
+  "_trn_potato_mode_title": "Modo PC potato",
+  "_trn_potato_mode_text": "¿Tienes un PC potato? ¡No te preocupes! Te tenemos cubierto. Habilita esto y todas las animaciones y efectos especiales se deshabilitarán.",
```
I would not use the "PC potato". This makes sense in english but in spanish is not an actual thing. 
For italian and japanese we have: "PC lento"
It could make sense to use that sentence instead of the direct translation from the english

MiguelProz replied:
In Spanish, there is also an expression borrowed from English, ‘PC Potato’, but I'll change it to a more colloquial use of the word.

MiguelProz committed:
Update potato mode titles and descriptions in Spanish

MiguelProz committed:
Merge branch 'develop' into develop

fulviodenza committed:
Merge branch 'develop' into develop

MiguelProz committed:
Merge branch 'develop' into develop

MiguelProz committed:
Actualiza las traducciones al español del ingles: agrega nuevas cadenas para apodos, avatares, y mejoras en la búsqueda de complementos.

MiguelProz committed:
Merge branch 'develop' into develop

MiguelProz committed:
Merge branch 'develop' into develop

MiguelProz commented:
Is there anything else needed to complete the pull request and upload the translation to dev?

kazaminosuke commented:
@MiguelProz🇪🇸

Soy japonés, así que no entiendo mucho español, pero he probado este PR localmente.
He confirmado que el español se ha añadido correctamente a las opciones de idioma.
¡Espero que se integre en una futura actualización!
(De parte del traductor de japonés de GDL)

I am Japanese, so I don't understand Spanish very well, but I tested this PR locally.
I confirmed that Spanish was correctly added to the language options.
I look forward to seeing this integrated in a future update!
(by GDL Japanese Translator)

私は日本人なのでスペイン語はあまりわかりませんが、このPRをローカルでテストしてみました。
言語オプションにスペイン語が正しく追加されていることを確認しました。
将来のアップデートで統合されることを楽しみにしています。
（GDL日本語翻訳者より）
<img width="1920" height="1129" alt="GDLauncher Carbon 2026_01_22 17_30_51" src="https://github.com/user-attachments/assets/2e53add0-8653-45be-b310-5abe5dc0dd9c" />

fulviodenza commented:
Hi @MiguelProz the PR looks good to me, let's just wait @blarfoon to merge it.

MiguelProz committed:
Merge branch 'develop' into develop

MiguelProz committed:
feat(i18n): update Spanish translations for various modules

- Removed nickname-related translations and replaced them with display name equivalents in auth.json.
- Added new shader loader setup messages in content.json.
- Introduced GDLPack entry in enums.json.
- Enhanced general.json with additional UI text for editing and dismissing actions.
- Expanded instances.json with server management and sharing functionalities.
- Updated java.json with memory warnings and migration instructions.
- Added search results and instance size options in library.json.
- Enhanced logs.json with server status and command input prompts.
- Expanded modals.json with new confirmation and cache cleanup messages.
- Updated notifications.json with server creation and EULA acceptance messages.
- Enhanced onboarding.json with instance sharing instructions.
- Improved search.json with additional server search context.
- Updated settings.json with cache storage management options.
- Enhanced tasks.json with detailed server installation tasks.
- Expanded ui.json with sorting and folder management options.

MiguelProz commented:
Updated to new modules. @blarfoon

blarfoon committed:
fix(i18n): polish Spanish strings

- restore {{needed}} placeholder in insufficient memory warning
- keep Inferno and Frost theme names untranslated like other locales
- standardize on "Shaders" over "Sombreadores"
- fix "Jump Back In" and "Star us on GitHub" wording

blarfoon committed:
Merge branch 'develop' into develop
